frontend/src/components/Posts.js

```diff
@@ -10,24 +10,27 @@ const Posts = props => {
   }, []);
 
   const fetchPosts = async () => {
-    const response = await fetch(
-      "http://localhost:5000/posts?_sort=id&_order=DESC"
-    ).catch(error => console.error("Error:", error));
+    const response = await fetch("/api/posts")
+      .catch(error =>
+        console.error("Error:", error)
+      );
 
     if (!response) {
       return;
     }
-
     const fetchedPosts = await response.json();
 
-    setPosts(fetchedPosts);
+    // TODO  setPosts(fetchedPosts);
```

Comment:
💃 Is this intentional?

Reply:
Yes, as we don't have a backend that returns posts at the moment. 
This feature will be implemented later on.